esy-lib/Tarball.re

```diff
@@ -67,6 +67,29 @@ let run = cmd => {
   };
 };
 
+let fixFilePermissionsAfterUnTar = out => {
+  Fs.traverse(
+    ~f=
+      (p, s) => {
+        let p = EsyBash.normalizePathForCygwin(Path.show(p));
+        try%lwt(
+          switch (s.st_kind) {
+          | Unix.S_REG =>
+            let%lwt () = Lwt_unix.chmod(p, 0o644);
+            Lwt.return(Run.return());
+          | Unix.S_DIR =>
+            let%lwt () = Lwt_unix.chmod(p, 0o755);
+            Lwt.return(Run.return());
+          | _ => Lwt.return(Run.return())
```

Comment:
I think we should also handle `S_LNK`

Reply:
I handled `S_LINK` same as files is that correct?

Reply:
Nope, confirmed, symlink permissions are meaningless on Linux, sorry for the noise